packages/components/src/PageHeader/PageHeader.tsx

```diff
@@ -16,7 +16,7 @@ import {
   Wrapper,
 } from './styles';
 
-export type PageHeaderProps = {
+export type PageHeaderProps<BackBtnProps = {}> = {
```

Comment:
```suggestion
export type PageHeaderProps<TBackButtonComponent extends ElementType = ElementType,
> = {
```
1. Можно описать типы, по аналогии с кнопкой, тогда не нужно будет явно указывать св-ва в дженерике
2. Параметры в дженерике должны иметь префикс T